cli/types/cypress-type-helpers.d.ts

```diff
@@ -1,2 +1,4 @@
 // type helpers
-type Nullable<T> = T | null
+declare namespace CyTypeHelpers {
```

Comment:
Generally we use `Cypress` as our namespace in our other `d.ts` files, can we do the same here?